msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -523,446 +38,9 @@ You can update `none` as a value for `authorization` in an API-based message ext
-
-```http
-GET /api/resource HTTP/1.1
-Host: api.example.com
-Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
-```
-
-After the API-based message extension gets a request header with token, perform the following steps:
-
-* **Authenticate**: Verify the token for the audience, scope, issuer, and signature claims to check if the token is for your app.
-
-  The following is an example of a JSON Web Token (JWT) with a header and response:
-
-  # [Token V2](#tab/token-v2)
-
-  ```json
-  {
-  "typ": "JWT",
-  "rh": "0.AhoAv4j5cvGGr0GRqy180BHbR6Rnn7s7iddIqxdA7UZsDxYaABY.",
-  "alg": "RS256",
-  "kid": "q-23falevZhhD3hm9CQbkP5MQyU"
-  }.{
-    "aud": "00000002-0000-0000-c000-000000000000",
-    "iss": "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0",
-    "iat": 1712509315,
-    "nbf": 1712509315,
-    "exp": 1712513961,
-    "aio": "Y2NgYEjJqF0stqv73u41a6ZmxPEvBgA=",
-    "azp": "1fec8e78-bce4-4aaf-ab1b-5451cc387264",
-    "azpacr": "0",
-    "name": "John Doe",
-    "oid": "00000000-0000-0000-0000-000000000000",
-    "preferred_username": "john.doe@contoso.com",
-    "rh": "I",
-    "scp": "access_as_user",
-    "sub": "e4uM7JgAEm08GBuasSltQjvPuMX1fR5TqxopJpqZJB8",
-    "tid": "12345678-aaaa-bbbb-cccc-9876543210ab",
-    "uti": "h7DMQwSPAEeiEe62JJUGAA",
-    "ver": "2.0"
-    }
-  ```
-
-  # [Token V1](#tab/token-v1)
-
-  ```json
-  {
-  "typ": "JWT",
-  "rh": "0.AhoAv4j5cvGGr0GRqy180BHbR6Rnn7s7iddIqxdA7UZsDxYaABY.",
-  "alg": "RS256",
-  "kid": "q-23falevZhhD3hm9CQbkP5MQyU"
-  }.{
-    "aud": "api://00000002-0000-0000-c000-000000000000",
-    "iss": "https://sts.windows.net/{tenantid}/",
-    "iat": 1537231048,
-    "nbf": 1537231048,
-    "exp": 1537234948,
-    "acr": "1",
-    "aio": "AXQAi/8IAAAA",
-    "amr": ["pwd"],
-    "appid": "c44b4083-3bb0-49c1-b47d-974e53cbdf3c",
-    "appidacr": "0",
-    "ipaddr": "192.168.1.1",
-    "name": "John Doe",
-    "oid": "00000000-0000-0000-0000-000000000000",
-    "scp": "access_as_user",
-    "sub": "AAAAAAAAAAAAAAAAAAAAAIkzqFVrSaSaFHy782bbtaQ",
-    "tid": "12345678-aaaa-bbbb-cccc-9876543210ab",
-    "uti": "fqiBqXLPj0eQa82S-IYFAA",
-    }
-  ```
-
-* **Use the token**: Extract the user information from the token, such as name, email, and object ID and use the token to call the message extension app's own API.
-
-  > [!NOTE]
-  > The API receives a Microsoft Entra token with the scope set to `access_as_user` as registered in the Azure portal. However, the token isn't authorized to call any other downstream APIs, such as Microsoft Graph.
-
-</details>
-<br/>
-
 ### Troubleshooting
```

Comment:
trouble shooting information has nothing to do with auth. Can you pls move it to a dedicated page ?

Reply:
Moved to troubleshooting article.